doc/pages/FAQ.md

```diff
@@ -62,7 +62,35 @@ remove) are also run in a sandbox and guaranteed not to affect your system.
 
 ---
 
-#### 🐫  Why does ``opam init`` need to add stuff to my init scripts / why is ``eval $(opam env)`` needed ?
+#### 🐫  Why does opam require ``bwrap`` installed ?
+
+Since opam 2.0.0~rc2, opam uses `bwrap` in Linux to run package instructions in
+a sandbox. See [bubblewrap page](https://github.com/projectatomic/bubblewrap)
+for details.
+
+These sandboxing instruction are specified in the built-in configuration, that
```

Comment:
s/instruction/instruction**s**

For what it's worth, "in the built-in configuration, as shown by `opam init --show-default-opamrc`:"